dsaas-services/fabric8-ui-admin-console.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+- hash: e1df2527134e449cbe2afb55231101a8c8538488
+  name: fabric8-ui-admin-console
+  path: /openshift/fabric8-admin-console.app.yaml
+  url: https://github.com/fabric8-ui/fabric8-admin-console/
+  hash_length: 6
```

Comment:
Please set `hash_length: 7`, that's the reason why this is failing.

Reply:
Updated